electron-app/src/server/websites/website.base.ts

```diff
@@ -37,6 +37,7 @@ export abstract class Website {
 
   abstract readonly BASE_URL: string;
   abstract readonly acceptsFiles: string[];
+  abstract MAX_CHARS: number;
```

Comment:
nit: if 99% are going to be -1, I feel like a readonly with default might be more convenient unless being explicit is that important here.

Reply:
Been mulling this over and I think being explicit on the max char count might be worthwhile; it's highly unlikely that these services actually are unlimited chars, and we just are missing a check for them - and we should slowly go back and fill them in ...